# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/D3D9SurfaceImage.h
# Commit: 6149f5b15c5e
# Full Hash: 6149f5b15c5e278ef15ea36fa896fa8c146ebdca
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2021-01-14 08:32:45
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1640607 - Send SurfaceDescriptors for GPU blitting for video-to-webgl. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101061
# ==============================================================================

diff -r 29fd3a6a7e27 -r 6149f5b15c5e gfx/layers/D3D9SurfaceImage.h
--- a/gfx/layers/D3D9SurfaceImage.h	Wed Jan 13 20:14:54 2021 +0000
+++ b/gfx/layers/D3D9SurfaceImage.h	Wed Jan 13 21:49:40 2021 +0000
@@ -93,8 +93,6 @@
                           const gfx::IntRect& aRegion);
 
   // Returns the description of the shared surface.
-  const D3DSURFACE_DESC& GetDesc() const;
-
   gfx::IntSize GetSize() const override;
 
   already_AddRefed<gfx::SourceSurface> GetAsSourceSurface() override;